Makefile

```diff
@@ -107,3 +115,7 @@ nginx-up:
 .PHONY: nginx-down
 nginx-down:
 	kubectl delete -f deploy/nginx.yaml
+
+push-minikube:
+	@docker save "$(SERVER_IMAGE):latest" | ssh -i "${SSH_KEY_FILE}" ${SSH_USER}@${K8S_NODE} docker load || exit 1
```

Comment:
I've never seen this approach before.. is it an alternative to running `eval $(minikube docker-env)`?